presto-tests/src/main/java/io/prestosql/tests/statistics/StatisticsAssertion.java

```diff
@@ -120,12 +119,6 @@ public Checks noEstimate(Metric metric)
             return this;
         }
 
-        public Checks noBaseline(Metric metric)
```

Comment:
Does it mean we should remove the NO_BASELINE enum option and throw here instead?
https://github.com/prestosql/presto/blob/c74b775e103a7a239aafef1b7ca15fe6370495fe/presto-tests/src/main/java/io/prestosql/tests/statistics/MetricComparison.java#L60-L61